packages/mobile/src/app/Drawers.tsx

```diff
@@ -120,12 +119,11 @@ const commonDrawersMap: { [Modal in Modals]?: ComponentType } = {
   VipDiscord: VipDiscordDrawer,
   ProfileActions: ProfileActionsDrawer,
   PlaybackRate: PlaybackRateDrawer,
-  // PremiumContent, AddFunds, PurchaseVendor, USDCManualTransfer, and StripOnRamp *must* be in this order
+  // PremiumContent, AddFunds, PurchaseVendor, and StripOnRamp *must* be in this order
   // to avoid zIndex issues.
   PremiumContentPurchaseModal: PremiumContentPurchaseDrawer,
   AddFundsModal: AddFundsDrawer,
```

Comment:
Nit but should we call this AddCashModal?

Reply:
will do in follow-up PR